namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStore.scala

```diff
@@ -212,6 +225,51 @@ class ConsulDtabStore(
         Future.Unit
       }
     }
-    Activity(run).stabilize
+    dtabStatus.putIfAbsent(ns, InstrumentedDtab(run, pollState))
+    Activity(run.underlying).stabilize
   }
+
+  val handlerPrefix = root.show.drop(1) // drop leading "/"
+
+  override def adminHandlers: Seq[Admin.Handler] = Seq(
+    Admin.Handler(
+      s"/namer_dtabs_store/${handlerPrefix}.json",
+      new ConsulDtabStoreHandler(dtabStatus.asScala.toMap)
+    )
+  )
 }
+
+private[consul] case class InstrumentedDtab(
+  act: InstrumentedVar[Activity.State[Option[VersionedDtab]]],
```

Comment:
I think we should also add a custom serializer for VersionedDtab so that this gets printed more nicely in the output.  In particular, we would want to serialize the version byte array as a string:

```
      "value": {
        "dtab": "/svc=>/#/io.l5d.fs;/svc/cat=>/#/io.l5d.fs/dog | /#/io.l5d.fs/cat",
        "version": "MjAw"
      }
```

Reply:
I shamelessly stole the one for k8s from @dadjeibaah (thanks)